src/en/editorial-content/general.md

```diff
@@ -2,48 +2,45 @@
-- Avoid layout and data tables.
-- Avoid information-conveying content in images (graphs, schemas, diagrams,…)
-- Signal the end of the document (for instance, an image saying “End of email”, or a white-on-white text “End of document”).
-- Use simple, clear and short phrase turns: subject, verb, complement. One idea per sentence, with simple words.
-- Explain complicated words.
-- Use the present tense, preferably direct style, and active form (no subjunctive, no conjunctions).
-- Use proper typographic characters: dash -, em-dash —, “rounded quotes”.
-<!-- removed 1 line re: accents and caps – not relevant in English --> 
-- Avoid foreign words, abbreviations except those that are genuinely used.
-
-## Provide accessible layout tables
-
-Simplify tables, even if we know that email clients (including Outlook) still often demand to use tables for layout.
-
-Checkpoints:
-
-- In general, don’t do complex table layouts (merging cells, multicolumn,…).
-- Avoid embedding layout tables (maximum two levels of embedding), but also:
-- Don’t merge table cells, but use several simpler tables.
-- Don’t keep empty cells.
-- Don’t add background images.
-- Make sure that the sequential order is correct (one can read left to right and top to bottom).
-- Avoid complex data tables, graphs or schemas.
+* Make sure [color is not the only means used to communicate information](/en/web/design/colors-and-contrasts/#do-not-use-colour-or-sensory-characteristics-as-the-unique-source-of-information).
+* Ensure a level of [sufficient contrast between the color of the text and that of the background](/en/web/design/colors-and-contrasts/#make-sure-there-is-enough-contrast-between-font-and-background-colors). This is valid for your texts but also for icons, buttons and other graphic elements. Contrast can be checked using the tool [Color Contrast Analyzer](https://www.paciellogroup.com/resources/contrastanalyser/) for example:
+  * 4.5:1 for normal size text.
+  * 3:1 for large text and interface components or graphical elements that convey information.
+
+## Make reading easier
+
+* Align texts to the left (avoid justifying).
+* Put a capital letter at the beginning of a sentence but avoid whole sentences in capital letters.
+* Avoid italics.
+* Use simple, sans serif fonts: Arial, Calibri or [accessible font Dfa](https://opensource.orange.com/fr/category/actualites/ow2-fr/osai-fr/accessible_dfa-fr/), having a size of at least 10 points.
+* Set up simple punctuation.
+* Avoid textured backgrounds (images).
+* Do not make repeated carriage returns or tabulation to space (but the “Paragraph & Spacing” or “Indent” function in Word, for example).
+* Avoid layout tables or complex data tables (except in HTML if the [table structure is accessible](/en/web/develop/textual-content/#structuring-data-tables)).
+* Avoid "image" content that conveys information: graphs, diagrams, diagrams (except in HTML if a [textual alternative is available](/en/web/design/non-textual-content/)).
```

Comment:
s/"image"/“image”/

Reply:
Fixed in c216677282df01aa3e077a45ded6d6e8e62eff4d